modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -8,11 +8,11 @@
 <%@ include file="/init.jsp" %>
 
 <%
-StyleBookDisplayContext styleBookDisplayContext = new StyleBookDisplayContext(request, liferayPortletRequest, liferayPortletResponse);
+StyleBookDisplayContext styleBookDisplayContext = (StyleBookDisplayContext)request.getAttribute(StyleBookDisplayContext.class.getName());
 %>
 
 <clay:management-toolbar
-	managementToolbarDisplayContext="<%= new StyleBookManagementToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, styleBookDisplayContext.getStyleBookEntriesSearchContainer()) %>"
+	managementToolbarDisplayContext="<%= (StyleBookManagementToolbarDisplayContext)request.getAttribute(StyleBookManagementToolbarDisplayContext.class.getName()) %>"
```

Comment:
You can get as attribute after styleBookDisplayContext and just pass the variable after, like this:

```
StyleBookDisplayContext styleBookDisplayContext = (StyleBookDisplayContext)request.getAttribute(StyleBookDisplayContext.class.getName());
StyleBookManagementToolbarDisplayContext styleBookManagementToolbarDisplayContext = (StyleBookManagementToolbarDisplayContext)request.getAttribute(StyleBookManagementToolbarDisplayContext.class.getName();
```
then
```
<clay:management-toolbar
	managementToolbarDisplayContext="<%= styleBookManagementToolbarDisplayContext %>"
```

Reply:
@algobob 
I made that change and when I run `gradlew formatSource` it reverts back